bluetooth-plane/index.html

```diff
@@ -0,0 +1,207 @@
+          <paper-button id="right" raised class="colorful">Right</paper-button>
+          <paper-button id="stop" raised class="colorful">Stop</paper-button>
+        </div>
+      </paper-card>
+
+      <paper-card>
+        <div class="card-content">
+          <div class="center horizontal layout">
+            <div>Engine</div>
+            <paper-slider id="engine" class="red" value="0" max="254" editable></paper-slider>
+          </div>
+        </div>
+      </paper-card>
+
+      <paper-card>
+        <div class="card-content">
+          <div class="center horizontal layout">
+            <div>Rudder</div>
+            <paper-slider id="rudder" class="red" value="0" min="-128" max="127" editable></paper-slider>
+          </div>
+        </div>
+      </paper-card>
+
+      <paper-dialog id="dialog">
+        <h2>Error</h2>
+        <p>Could not connect to bluetooth device!</p>
+      </paper-dialog>
+    </div>
+
+    <platinum-bluetooth-device services-filter='["f000ffc0-0451-4000-b000-000000000000"]' id="plane-device">
+      <platinum-bluetooth-characteristic id="engineCharacteristic"
+        service="75b64e51-f181-4ed1-921a-476090d80ba7"
+        characteristic="75b64e51-f184-4ed1-921a-476090d80ba7"
+        auto-write>
+      </platinum-bluetooth-characteristic>
+      <platinum-bluetooth-characteristic id="rudderCharacteristic"
+        service="75b64e51-f181-4ed1-921a-476090d80ba7"
+        characteristic="75b64e51-f185-4ed1-921a-476090d80ba7"
+        auto-write>
+      </platinum-bluetooth-characteristic>
+    </platinum-bluetooth-device>
+
+    <script>
+      document.addEventListener('WebComponentsReady', function() {
+        var progress = document.querySelector('#progress');
+        var dialog = document.querySelector('#dialog');
+        var engine = document.querySelector('#engine');
+        var rudder = document.querySelector('#rudder');
+        var leftButton = document.querySelector('#left');
+        var rightButton = document.querySelector('#right');
+        var centerButton = document.querySelector('#center');
+        var stopButton = document.querySelector('#stop');
+        var planeDevice = document.querySelector('#plane-device');
+        var engineCharacteristic = document.querySelector('#engineCharacteristic');
+        var rudderCharacteristic = document.querySelector('#rudderCharacteristic');
+        progress.hidden = true;
+
+        function handleError(error) {
+          console.log(error);
+          progress.hidden = true;
+          dialog.open();
+        }
+
+        engine.addEventListener('value-change', function() {
+          progress.hidden = false;
+          planeDevice.request().then(function() {
+            engineCharacteristic.value = new Uint8Array([engine.value]);
+            progress.hidden = true;
+          })
+          .catch(handleError);
+        });
+        engine.addEventListener('platinum-bluetooth-auto-write-error', handleError);
```

Comment:
Looking at the polymer component code, agree this should be attached to characteristic.